frontend/projects/upgrade/src/assets/i18n/en.json

```diff
@@ -382,6 +382,9 @@
   "segments.duplicate-segment.segmentName-input-placeholder.text": "Name of the duplicate segment",
   "segments.duplicate-segment.segmentDescription-input-placeholder.text": "Description of the duplicate segment",
   "segments.duplicate-segment-duplicate.text": "duplicate",
+  "segments.overview.export-segment.text": "Export Segment",
+  "segments.overview.export-warning.text": "The segment data will be sent to '{{ email }}'.",
```

Comment:
Does this 'warning' text ever get used? It looks like the segment CSV downloads are not emailed.

Reply:
we were but it was unnecessary code, thx for pointing it out